gci/view_overview.py

```diff
@@ -0,0 +1,57 @@
+from django.http import HttpResponse
+import requests
+import json
+
+from .students import get_students, get_linked_students
+from .gitorg import get_logo
+from gci.config import get_api_key
+
+
+def index(request):
+    try:
+        client = get_api_key('GCI')
+        api_key_available = True
+    except BaseException:
+        client = None
+        api_key_available = False
+
+    s = []
+
+    if api_key_available:
+        linked_students = list(get_linked_students(get_students()))
+        org_name = linked_students[0]['organization_name']
+        favicon = get_logo(org_name, 16)
+        with open('_site/favicon.png', 'wb') as favicon_file:
+            favicon_file.write(favicon)
+
+        org_logo = get_logo(org_name)
+        with open('_site/org_logo.png', 'wb') as org_logo_file:
+            org_logo_file.write(org_logo)
+        s.append('<link rel="shortcut icon" type="image/png" '
+                 'href="../static/favicon.png"/>')
+        s.append('<img src="../static/org_logo.png" alt="'+org_name+'">')
+    else:
+        s.append(
+            'WARNING: API key not available. Expect limited functionality')
+
+    s.append('<title>Community website</title>')
+    s.append('<ul><li><a href="/gci/">Google Code-in</a>'
+             '<li><a href="/activity/">GitHub activity</a></ul>')
+
+    if api_key_available:
+        api_data_dump = json.loads(
+            requests.get('https://gci-leaders.netlify.com/data.json').content)
+        for item in api_data_dump:
+            if item['name'] == org_name:

```

Comment:
The new twitter component can load `org_name.txt` which is created at https://github.com/coala/community/blob/master/.ci/build.sh#L5 and https://github.com/coala/community/blob/master/orgname.sh#L5

Then it does not need `org_name = linked_students[0]['organization_name']` and all that mess I wrote last week (I am rewriting it!)

Reply:
that's useful, so the same can be done with the org logo at some stage so that it isn't reliable on api access